pkgs/tools/misc/dashing/default.nix

```diff
@@ -21,6 +21,6 @@ buildGoPackage rec {
     homepage    = https://github.com/technosophos/dashing;
     license     = licenses.mit;
     maintainers = [ ];
-    platforms   = platforms.linux;
+    platforms   = platforms.linux ++ platforms.darwin;
```

Comment:
Is there any reason for this not to be `platforms.all`?

Reply:
it's go so probably not 😄